CONFIGURATION.md

```diff
@@ -2,18 +2,18 @@
 
 #### Creating a superuser
 
-*Is assumed that by this time all images are running on docker. (check: - [Running the application in local environment](#running-the-application-in-local-environment))
+*This should be executed after executing all docker commands. (check: - [Running the application in local environment](#running-the-application-in-local-environment))
 
 1. Create a superuser for admin access running `python manage.py createsuperuser`.
-2. Go to `http://localhost:8010/admin` and log in with the superuser credentials.
+2. Go to `http://localhost:8010/admin` and sign in with the superuser credentials.
 
 #### AWS Services Configuration in the Admin Web
 
-*In order to apply this configurations, login to the admin web is needed.
+*In order to apply the following configurations, signing in to admin web is needed.
 
-1. **CloudFront Configuration:** Inside the Global configuration, apply the settings located in configuration/configuration.samples/cloud_front_configuration.sample on the CloudFront Configuration input.
-2. **MediaConvert Configuration:** Create a `MediaConvert Configuration` named `default`, using the settings found in `configuration/configuration.samples/media_convert_configuration.sample`.
-3. **MediaLive Configuration:** Create `MediaLive Configuration`, also named `default`, and apply the following configurations:
+1. **CloudFront Configuration:** Inside `Global configuration`, apply the settings located in `configuration/configuration.samples/cloud_front_configuration`. on the `CloudFront Configuration` input.
+2. **MediaConvert Configuration:** Create a `MediaConvert Configuration` and add a name to it, using the settings found in `configuration/configuration.samples/media_convert_configuration.sample`.
+3. **MediaLive Configuration:** Create a `MediaLive Configuration`, add a name to it, and apply the following configurations:
    - **Input Configuration:** `configuration/configuration.samples/media_live_input_attachments.sample`.
    - **Destination Configuration:** `configuration/configuration.samples/media_live_destinations.sample`.
    - **Encoder Configuration:** `configuration/configuration.samples/media_live_encoder_settings_economic.sample`.
```

Comment:
Here I explain that we have those samples (and we already created the default ones with those sample files, but if you want to modify those files you can then find them under configuration/x... file and expand/change those "default" ones to then create more configurations